### PR TITLE
Uses Lograge for logging, so the log output style is the same as in Champaign and will hit the same log filters on Papertrail.

### PR DESCRIPTION
**Pull request by Tuuleh**

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -44,9 +44,19 @@
...
-  config.log_level = :debug
+  config.log_level = :info
+  config.lograge.enabled = true
+
+  config.lograge.custom_options = lambda do |event|
+    params = event.payload[:params].reject do |k|
+      ['controller', 'action'].include? k
+    end
+    log_hash = {"params"=> params, "time"=>event.time}
+    if not event.payload[:exception].blank?
```
Comment (NealJMD) [inline config/environments/production.rb]: 1) don't use ruby `not`, use `!` ([source](https://github.com/bbatsov/ruby-style-guide#bang-not-not)) (basically cause it doesn't have the same binding priority)
2) here just use `if event.payload[:exception].present?`

File: config/environments/production.rb [hunk 2]
```diff
@@ -44,9 +44,19 @@
...
+      log_hash["exception"]=event.payload[:exception]
```
Comment (NealJMD) [inline config/environments/production.rb]: also you don't have to go fix this but try to keep spaces around your equal signs

File: config/environments/production.rb [hunk 3]
```diff
@@ -44,9 +44,19 @@
...
 
-  # Use the lowest log level to ensure availability of diagnostic information
-  # when problems arise.
-  config.log_level = :debug
+  config.log_level = :info
+  config.lograge.enabled = true
+
+  config.lograge.custom_options = lambda do |event|
+    params = event.payload[:params].reject do |k|
+      ['controller', 'action'].include? k
```
Comment (NealJMD) [inline config/environments/production.rb]: why don't we want to include controller and action? just curious

Comment (NealJMD) [inline config/environments/production.rb]: also to confirm, this loop implies that the event payload has symbol keys at one level and string keys at another, ie you would read things as `event.payload[:params]['controller']`. is that right?

Comment (Tuuleh) [inline config/environments/production.rb]: This code includes controller and action - they're not included without this. It's the same as we have in Champaign at the moment. `reject` returns a copy of the params hash, comparing them to `['controller',  'action']` allows to keep controller and action from the copy of the hash. This example is actually used in the [Lograge repository](https://github.com/roidrage/lograge), but to address your concerns - it does work as expected: 
```
Feb 24 13:01:14 ak-worker-staging production.log:  I, [2016-02-24T21:01:14.317817 #8]  INFO -- : method=POST path=/message format=html controller=message_handler action=handle status=200 duration=2574.18 view=0.16 db=2.91 params={"type"=>"update_share", "button_id"=>24, "message_handler"=>{"type"=>"update_share", "button_id"=>24}} time=2016-02-24 21:01:11 +0000
```

Comment (NealJMD) [inline config/environments/production.rb]: OK I get it - it's already logging controller and action, so no need to include it in params. looks good!

**Pull request merged**